include/gsl/span

```diff
@@ -191,6 +191,11 @@ namespace details
             return ret += n;
         }
 
+        friend constexpr span_iterator operator+(difference_type n, span_iterator const& rhs)
+        {
+            return rhs + n;
+        }
+
         constexpr span_iterator& operator+=(difference_type n)
```

Comment:
`<iterator> + <difference>` and `<difference> + <iterator>` are equivalent expressions since `+` is commutative. Since `-` is *not* commutative, `<difference> - <iterator>` has no defined meaning and is therefore not required.